site/data/kapp-controller/docs/kapp-controller-develop-toc.yml

```
@@ -27,6 +27,8 @@ toc:
         url: /packaging-tutorial
       - page: "Consuming Packages using the CLI"
         url: /kctrl-package-tutorial
+      - page: Authoring packages with the CLI
```

```suggestion
      - page: Authoring packages using the CLI
```
It will be consistent with `Consuming Packages using the CLI`